src/test/java/com/ipt/kafkatopicupdates/streams/DefaultStreamsTopologyTest.java

```diff
@@ -0,0 +1,270 @@
+
+package com.ipt.kafkatopicupdates.streams;
+
+import ch.ipt.kafka.avro.Authorization;
+import com.ipt.kafkatopicupdates.StreamsUtils;
+import io.confluent.kafka.streams.serdes.avro.SpecificAvroSerde;
+import org.apache.kafka.common.serialization.Serde;
+import org.apache.kafka.common.serialization.Serdes;
+import org.apache.kafka.streams.*;
+
+import org.apache.kafka.streams.state.KeyValueStore;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+import java.util.Properties;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+
+
+class DefaultStreamsTopologyTest {
+
+    private DefaultStreamsTopology defaultStreamsTopology;
+
+    private Properties config;
+    private Map<String, Object> serdeConfig;
+    private TopologyTestDriver testDriver;
+
+    private String sourceTopic = "authorizations-with-duplicates";
+    private String sinkTopic = "authorizations-without-duplicates";
+    private String STORE_NAME = "test-store";
+
+    private TestInputTopic<String, Authorization> inputTopic;
+    private TestOutputTopic<String, Authorization> outputTopic;
+
+    private final Serde<String> stringSerde = Serdes.String();
+    private SpecificAvroSerde<Authorization> authorizationSerde;
+
+
+
+    @BeforeEach
+    public void setUp() {
+        config = new Properties();
+        config.put(StreamsConfig.APPLICATION_ID_CONFIG, "test");
+
+        config.put("schema.registry.url", "mock://test");
+
```

Comment:
Why not extract the Properties' creation to the Util class?

Reply:
Good idea